### PR TITLE
Redirect after confirming account

### PR DESCRIPTION
**Pull request by orangejenny**
http://manage.dimagi.com/default.asp?176522

On success, redirect to dashboard with a message:
screen shot 2015-07-15 at 6 24 05 pm

On error, show the same page as previously:
screen shot 2015-07-15 at 6 25 28 pm

@czue / @nickpell

### CODE REVIEW

File: corehq/apps/registration/views.py [hunk 1]
```diff
@@ -310,18 +304,12 @@ def confirm_domain(request, guid=None):
...
-    context['message_subtitle'] = _(
-        'Thank you for activating your account, %s!'
-    ) % requesting_user.first_name
-    context['message_body'] = _(
-        'Your account has been successfully activated.  Thank you for taking '
-        'the time to confirm your email address: %s.'
-    ) % requesting_user.username
-    context['is_error'] = False
-    context['valid_confirmation'] = True
+    messages.success(request,
```
Comment (dimagimon) [inline corehq/apps/registration/views.py]: W291 trailing whitespace

File: corehq/apps/registration/templates/registration/confirmation_complete.html [hunk 2]
```diff
@@ -1,34 +0,0 @@
-{% extends "registration/base.html" %}
-
-{% if valid_confirmation %}
-{% block js-inline %}{{ block.super }}
-<script>
-    {% if not is_error %}
-    $(function () {
-        analytics.workflow("Confirmed new project");
```
Comment (orangejenny) [inline corehq/apps/registration/templates/registration/confirmation_complete.html]: @amsagoff note that we're losing this bit of analytics

Comment (czue) [inline corehq/apps/registration/templates/registration/confirmation_complete.html]: if we wanted we could push this to python, right? not sure how much of a priority is.

File: corehq/apps/registration/templates/registration/confirmation_error.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+{% block title %}Error Confirming Account{% endblock title %}
+
+{% block step-title %}
+    {{ message_title }}{% if message_subtitle %} <small>{{ message_subtitle }}</small>{% endif %}
+{% endblock %}
+
+{% block registration-content %}
+    {% if message_body %}<p>{{ message_body }}</p>{% endif %}
+    <p>You may try <a href="{% url "registration_resend_domain_confirmation" %}">resending the confirmation email</a>.</p>
+    <p>In the meantime, please explore the <a href="/a/public/">CommCare HQ Example Project.</a></p>
```
Comment (czue) [inline corehq/apps/registration/templates/registration/confirmation_error.html]: @amsagoff do we still send people here? i didn't even know this thing still existed

**Comment by amsagoff** [comment]
@orangejenny confirming that they will normally still be logged in when they click the confirmation link, so they won't be redirected to the login page most of the time.

The demo project is definitely old and we don't redirect people anymore. Here's the current confirmation page, which directs people to the Exchange instead. I think this is still a fine page on error:

image

@orangejenny instead of "missing activation key" can we change the language to be more user-facing -- e.g. "Account not activated -- email not yet confirmed"

**Comment by czue** [comment]
looks good to me pending @amsagoff's requests. will let @nickpell do final review / merge

File: corehq/apps/registration/templates/registration/confirmation_complete.html (hunk 2 above)
Comment (orangejenny) [inline corehq/apps/registration/templates/registration/confirmation_complete.html]: yes, fair point, added them back in server-side

**Comment by orangejenny** [comment]
@amsagoff Yeah, that wasn't the best screenshot choice. Most of the time, people will already be logged in and will go the dashboard.

Replaced demo project link with exchange link, updated error text.

**Comment by orangejenny** [comment]
@nickpell should be g2g

**Pull request merged**